web/assets/dark.scss

```diff
@@ -0,0 +1,79 @@
+$sirius-bg-color: #29292e;
+
+.app-\!-html-class--dark {
+  background-color: $sirius-bg-color;
+  color: govuk-colour("white");
+  color-scheme: dark;
+
+  body.govuk-template__body,
+  .govuk-template {
+    background-color: $sirius-bg-color;
+    color: govuk-colour("white");
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  .govuk-heading-xl,
+  .govuk-heading-l,
+  .govuk-heading-m,
+  .govuk-heading-s,
+  .govuk-heading-xs,
+  p,
+  .govuk-list {
+    color: govuk-colour("white") !important;
+  }
+
+  .govuk-link:link {
+    color: lighten(govuk-colour("light-blue"), 20%);
```

Comment:
This isn't beautiful, but it ensures we're:
- Basing our colours on the Design System's core colour scheme
- Using colours which sufficiently contrast with the background